ArcGISRuntimeSDKQt_QMLSamples/Maps/GenerateOfflineMapLocalBasemap/README.md

```diff
@@ -0,0 +1,57 @@
+# Generate offline map with local basemap
+
+Use the `OfflineMapTask` to take a web map offline, but instead of downloading an online basemap, use one which is already on the device.
+
+![](screenshot.png)
+
+## Use case
+
+There are a number of use-cases where you may wish to use a basemap which is already on the device, rather than downloading:
+
+* You want to limit the total download size.
+* You want to be able to share a single set of basemap files between many offline maps.
+* You want to use a custom basemap (for example authored in ArcGIS Pro) which is not available online.
+* You do not wish to sign into ArcGIS.com in order to download Esri basemaps.
+
+The author of a web map can support the use of basemaps which are already on a device by configuring the web map to specify the name of a suitable basemap file. This could be a basemap which:
+
+* Has been authored in ArcGIS Pro to make use of your organizations custom data.
+* Is available as a PortalItem which can be downloaded once and re-used many times.
+
+## How to use the sample
+
+1. Click on "Generate Offline Map".
+2. You will be prompted to choose whether you wish to download the online basemap or use the "naperville_imagery.tpk" basemap which is already on the device.
+3. If you choose to download the online basemap, the offline map will be generated with the same (topographic) basemap as the online web map.
+4. To download the Esri basemap, you may be prompted to sign in to ArcGIS.com.
+5. If you choose to use the basemap from the device, the offline map will be generated with the local imagery basemap. The download will be quicker since no tiles are exported or downloaded.
+6. Since the application is not exporting online ArcGIS Online basemaps you will not need to log-in.
+
+## How it works
+
+The sample creates a `PortalItem` object using a web map's ID. This portal item is used to initialize an `OfflineMapTask` object. When the button is clicked, the sample requests the default parameters for the task, with the selected extent, by calling `OfflineMapTask::createDefaultGenerateOfflineMapParameters`.
```

Comment:
`OfflineMapTask.createDefaultGenerateOfflineMapParameters` for Qml? I'm not sure if that translates better, but this one is inconsistent with the rest.